db/models.py

```diff
@@ -0,0 +1,24 @@
+from sqlalchemy import Column, Integer, String, Date, ForeignKey, Text
+from sqlalchemy.orm import relationship
+
+from db.engine import Base
+
+
+class Author(Base):
+    __tablename__ = "author"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(100), nullable=False, unique=True)
+    bio = Column(Text)
+
+
+class Book(Base):
+    __tablename__ = "book"
+
+    id = Column(Integer, primary_key=True, index=True)
+    title = Column(String(100), nullable=False, unique=True)
```

Comment:
The `title` column in the `Book` model is marked as `unique=True`. Consider whether this constraint is appropriate, as it may not allow multiple books with the same title, which could be a valid scenario.